www.example.com/db/openid-store/associations/http-localhost-Sve98kiqQxHfzN8CdZK0Uuu1aIs-79QTAXpF_XIxzDrDmpcZxJStiSQ

```diff
@@ -0,0 +1,6 @@
+version:2
```

Comment:
Ah, and these two files also don't need to be included. I'll modify our `.gitignore` to not detect these anymore... thanks, but in the meantime would you mind removing them too?